src/setup/scss/extends/_invisible.scss

```diff
@@ -0,0 +1,8 @@
+%invisible {
```

Comment:
please change to be a mixin, as we are getting rid of any extends in the PR #182 because extends are not supported with @use and @forward